cmd/virtblocks/main.go

```diff
@@ -0,0 +1,18 @@
+package virtblocks
+
+import (
+	"fmt"
+	"github.com/virtblocks/virtblocks/pkg/devices"
+	"github.com/virtblocks/virtblocks/pkg/util"
```

Comment:
Is this the usual naming convention for Go projects? Eg. `pkg/` for the library part, and `cmd/` for programs that make use of it? Sounds good, just making sure I understood it correctly.

What about the `virtblocks/` in `cmd/virtblocks/`? It seems redundant. Is that also part of the usual naming convention?

Reply:
Normally you have a `cmd` folder and then  directories underneath it with the names of the binaries. That was just the first name I could think about.

Reply:
And yes `pkg` and `cmd` are pretty much standrad.